### DISCUSSION TITLE
Is the Tweepy project still alive?

### DISCUSSION THREAD

**Discussion by SQLGUY70** [tweepy/tweepy#2124] [Questions] [answered]
It is currently June 3, 2023... I don't see any updates since April.  Is this project still alive?

Elon has really jacked things up... My scripts, using Tweepy, are really having a hard time.  Is this the end?

Thanks all!

-Mark

**Comment by Harmon758** [comment, marked as answer]
There have been no API changes requiring any updates to the library since then.
The last commit was April 24. A month or so of no commits is hardly the end or death of a stable library. In fact, Tweepy has had much longer periods of inactivity in the past.

Duplicate of #2100
